### PR TITLE
FIX - Responsive design for the detail page

### PR DESCRIPTION
**Pull request by DesleyAalderink**
I made the responsive design for the detail page. 
Let me know what you think.

### CODE REVIEW

File: src/styles/style.css [hunk 1]
```diff
@@ -360,6 +361,7 @@
 
 .detailInformation {
   width: 40%;
+  /* max-width: 40%; */
```
Comment (muise001) [inline src/styles/style.css]: remove all unused CSS please

File: src/styles/style.css [hunk 2]
```diff
@@ -462,10 +465,75 @@
...
+    width: 100%;
+    padding: 7rem 7rem 5rem 7rem;
+  }
+
+  .detailInformation h1 {
+    font-size: calc(100vw*0.09);
+  }
+
+  .detailInformation p:nth-of-type(1) {
+    font-size: calc(100vw*0.04);
```
Comment (muise001) [inline src/styles/style.css]: Nice calc's!

File: src/styles/style.css [hunk 3]
```diff
@@ -462,10 +465,75 @@
...
+    font-size: 2.3em;
+  }
+
+  #detailRecommendation {
+    position: initial;
+  }
+
+  .detailCover img {
+    width: 100%;
+    height:100%;
```
Comment (muise001) [inline src/styles/style.css]: to size an image you can ether use with OR height. You don't have to style both. (also missing a space after the ":")

File: src/styles/style.css (hunk 1 above)
Comment (DesleyAalderink) [reply src/styles/style.css]: you got that aswell, hypocrite

Comment (muise001) [reply src/styles/style.css]: wanna go M8? 💢

**Pull request merged**